exp-player/addon/components/exp-rating-form/component.js

```diff
@@ -378,13 +380,14 @@ var questions = [
                 scale: SEVEN_POINT_SCALE,
                 value: null,
                 labelTop: false,
+                formatLabel: 'measure-six',
                 labels: [
                     {
                         rating: 0,
```

Comment:
I was mapping the labels to the correct radio button option based on index - it looks odd since some of the scales start at 1 and not 0. I could make this more explicit by comparing the value instead of the index.